cli/internal/cloudcmd/upgrade.go

```diff
@@ -112,6 +112,17 @@ func (u *Upgrader) updateMeasurements(ctx context.Context, newMeasurements measu
 		return nil
 	}
 
+	// don't allow potential security downgrades by setting the warnOnly flag to true
+	for k, newM := range newMeasurements {
+		currentM, ok := currentMeasurements[k]
+		if !ok {
+			continue
+		}
+		if !currentM.WarnOnly && newM.WarnOnly {
```

Comment:
```suggestion
		if currentM, ok := currentMeasurements[k]; ok && !currentM.WarnOnly && newM.WarnOnly {
```
or keep it as is if you find this one too hard to read